src/modules/profile/api/profileInfo.ts

```diff
@@ -52,5 +59,6 @@ export function mapProfileInfo({
     instagram: data.instagram,
     twitter: data.twitter,
     facebook: data.facebook,
+    isVerified: data.identity === VerifiedsState.Verified ? true : false,
```

Comment:
//  bad
data.identity === VerifiedsState.Verified ? true : false

// Modify to
data.identity === VerifiedsState.Verified